src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[6];
+
+        for (int i = 0; i < 3; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+        }
+
+        for (int i = 0; i < 6; i++) {
+            figures[i] = figureSupplier.getDefaultFigure();
```

Comment:
You should create several random Figures using a loop, but you should not create a new `FigureSupplier` instance inside the loop. However, in this case, you are correctly using the same `figureSupplier` instance outside the loop, which is good.